packages/network-explorer/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "private": true,
+  "type": "module",
+  "scripts": {
+    "dev": "nodemon",
```

Comment:
You could also just call `tsx server.ts` or `pm2` or some other daemon thing here